openqa-maintenance.py

```diff
@@ -79,6 +79,12 @@
 #
 #        return [p for p in packages if p.name in in_channel]
 
+with open("./osc-plugin-factory/data/kgraft.json", 'r') as f:
+    KGRAFT_SETTINGS = json.load(f)
```

Comment:
please do not hardcode the checkout path. Better access a relative path __file__ - https://stackoverflow.com/a/595332